tests/integration/utils/parseXML.js

```diff
@@ -0,0 +1,22 @@
+/*
+*   Parse XML into a document
+*   @param {Object} text: string to convert
+    @return {Object} xmlDoc: the converted document
+*/
+export default function parseXML(text) {
```

Comment:
No need to have this util. DOMParser seems to exist in every target we have:
https://developer.mozilla.org/en-US/docs/Web/API/DOMParser#Browser_compatibility
http://caniuse.com/#feat=xml-serializer